.gitlab-ci.yml

```diff
@@ -98,6 +101,9 @@ test:acceptance:
       --mender-connect-deb-version $(cat ${CI_PROJECT_DIR}/output/mender-connect-deb-version)
       --mender-configure-version $MENDER_CONFIGURE_VERSION
       --mender-configure-deb-version $(cat ${CI_PROJECT_DIR}/output/mender-configure-deb-version)
+      --mender-monitor-version $MENDER_MONITOR_VERSION
+      --mender-monitor-deb-version $(cat ${CI_PROJECT_DIR}/output/mender-monitor-deb-version)
+      --commercial-tests $(echo "${MENDER_PRIVATE_REPO_ACCESS_TOKEN}")
```

Comment:
Please don't - let's find a better way. You could check if the variable is not empty, then set another variable to true. Read [LFF from 17-02-2021](https://script.google.com/a/northern.tech/macros/s/AKfycbxgcRZg4TNHuCwoLJNBK6FWQR3qfUBSNDVViBDTf7ebM4vsSRow/exec) (Search for that date)